cbdr/cortex
# COR-736: Error Message on New Fields

## Purpose:
Adds JS for required fields, making sense of what's being passed over from the plugin as of [#56](https://github.com/cortex-cms/cortex-plugins-core/pull/56)

> As a writer, when I create a new blog from the dashboard, I automatically see the red field errors before I've even begun.

## JIRA:
https://cb-content-enablement.atlassian.net/browse/COR-736

## Steps to Take On Prod
* Go to the form to create a new Employer Blog
* See that there are no red fields on form load

## Changes:
* Changes to setup
  * Will need to bump `cortex-plugins-core` to the changes seen in [#56](https://github.com/cortex-cms/cortex-plugins-core/pull/56)

* Architectural changes
  * N/A

* Migrations
  * N/A

* Library changes
  * [#56](https://github.com/cortex-cms/cortex-plugins-core/pull/56)

* Side effects
  * N/A

## Screenshots
* Before
<img width="1280" alt="screen shot 2017-04-21 at 10 57 35 am" src="https://cloud.githubusercontent.com/assets/8419757/25285868/60862c3e-2681-11e7-82e4-923979ca5115.png">

* After
<img width="1280" alt="screen shot 2017-04-21 at 10 57 06 am" src="https://cloud.githubusercontent.com/assets/8419757/25285872/63e005a8-2681-11e7-83d6-795a1801cebc.png">

## QA Links:
http://web.cortex-6.development.c66.me/

## How to Verify These Changes
* Specific pages to visit
  * Create an Employer Blog

* Steps to take
  * See there are no red fields

* Responsive considerations
  * N/A

## Relevant PRs/Dependencies:
[#56](https://github.com/cortex-cms/cortex-plugins-core/pull/56)

## Additional Information
N/A
